src/GrpcProtos/capture.proto

```diff
@@ -749,7 +766,7 @@ message ClientCaptureEvent {
     // use them for high frequency events. For the rest please assign
     // numbers starting with 16.
     //
-    // Next high-frequency ID: 12
+    // Next high-frequency ID: 13
```

Comment:
Do I get it right that this event should happen with the same frequency as frame rate? If yes, that's very low frequency, so could you change to use one of the "low-frequency" ids before the branch cut?